Cristofer Caballero y Manuel Rosario/problema1parcial2.c

```diff
@@ -0,0 +1,24 @@
+#include<stdio.h>
+void main()
+{
+	int num=0, a=0,b=0,c=0,d=0;
+	for(a=2; a<=25; a++){
+		if(a%2==0)
+		printf("%d\t",a);
+	}
+	printf("\n");
+	for(b=3; b<=25; b++){
+		if(b%3==0)
+		printf("%d\t",b);
+	}
+	printf("\n");
+		for(c=4; c<=25; c++){
+		if(c%4==0)
+		printf("%d\t",c);
+	}
+	printf("\n");

```

Comment:
Falta de 5 en 5, de 6 en 6, .... hasta 25 en 25... ese indice que tienes un multiples for lo hubiera podido manejar con otro ciclo y hubiera sido codigo mas limpio y corto